EPRN.Portal/Controllers/WasteController.cs

```diff
@@ -237,9 +237,8 @@ public async Task<IActionResult> BaledWithWire(int? id)
             if (id == null)
                 return NotFound();
 
-            var model = await _wasteService.GetBaledWithWireModel(id.Value);
-            if (model.BaledWithWireDeductionPercentage == null)
-                model.BaledWithWireDeductionPercentage = _homeService.GetBaledWithWireDeductionPercentage();
+            double? deductionAmountFromConfig = _homeService.GetBaledWithWireDeductionPercentage();
+            var model = await _wasteService.GetBaledWithWireModel(id.Value, deductionAmountFromConfig.GetValueOrDefault());
```

Comment:
The config amount should be injected into the waste service, and therefore will not need to have it passed as a parameter

Reply:
Done.